providers/netty4/src/main/java/org/asynchttpclient/netty/handler/Processor.java

```diff
@@ -71,11 +75,41 @@ public void channelRead(final ChannelHandlerContext ctx, Object msg) throws Exce
                 ac.call();
                 Channels.setDiscard(channel);
             }
+            ReferenceCountUtil.release(msg);
+
 
         } else if (attribute instanceof NettyResponseFuture) {
             NettyResponseFuture<?> future = (NettyResponseFuture<?>) attribute;
             protocol.handle(channel, future, msg);
 
+        } else if (attribute instanceof StreamedResponsePublisher) {
```

Comment:
@jroper Isn't this https://github.com/dotta/async-http-client/blob/issue/544-reactive-streams-support/providers/netty4/src/main/java/org/asynchttpclient/netty/handler/Processor.java#L158-L159 already taking care of retrying for the first condition. If not, could you or @slandelle give me a head start on this.